commands/SingleOp.java

```diff
@@ -1,22 +1,26 @@
 package org.usfirst.frc4904.standard.commands;
 
+import edu.wpi.first.wpilibj2.command.CommandBase;
 
-import edu.wpi.first.wpilibj.command.Command;
-
-public class SingleOp extends Command {
+public class SingleOp extends CommandBase {
 	protected Runnable op;
 
-	public SingleOp(Runnable op) {
+	public SingleOp(String name, Runnable op) {
+		setName(name);
```

Comment:
do you want to call `super()` here?

Reply:
I guess? I think somebody told me to remove `super()` calls earlier.